src/core/pusher.ts

```diff
@@ -180,18 +180,28 @@ export default class Pusher {
     }
   }
 
-  bind(event_name : string, callback : Function) : Pusher {
-    this.global_emitter.bind(event_name, callback);
+  bind(event_name : string, callback : Function, context? : any) : Pusher {
+    this.global_emitter.bind(event_name, callback, context);
     return this;
   }
 
-  unbind(event_name: string, callback: Function) : Pusher {
+  unbind(event_name? : string, callback? : Function, context? : any) : Pusher {
```

Comment:
What I meant is that your change added the parameter to the function but the parameter itself remains unused. I assume that you intended to pass it to `this.global_emitter.unbind`, which after taking a look into the Dispatcher appears correct.